.github/workflows/production.yml

```diff
@@ -61,3 +61,16 @@ jobs:
           K8S_SECRET_DATABASE_DB: ${{ secrets.K8S_SECRET_DATABASE_DB_STABLE }}
           K8S_SECRET_DATABASE_USERNAME: ${{ secrets.K8S_SECRET_DATABASE_USERNAME_STABLE }}
           K8S_SECRET_DATABASE_PASSWORD: ${{ secrets.K8S_SECRET_DATABASE_PASSWORD_STABLE }}
+
+      - name: Deploy Publish Scheduled Pages Cronjob
+          uses: City-of-Helsinki/setup-cronjob-action@main
+          with:
+            image_repository: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}
+            image_tag:  ${{ github.sha }}
+            secret_name: "project-events-helsinki-cms-secret"
+            name: "publish-scheduled-pages"
+            kubeconfig_raw: ${{ secrets.KUBECONFIG_RAW_STABLE}}
+            target_namespace: ${{ secrets.K8S_NAMESPACE_STABLE }}
+            schedule: '* * * * *' # once per 1 minute
+            command: "{/app/manage.py}"
+            args: "{publish_scheduled_pages}"
```

Comment:
Not sure if this works like this. See e.g. https://github.com/City-of-Helsinki/kukkuu/blob/develop/.github/workflows/staging.yml#L86-L87 which has working version.

Testing is also possible:
https://github.com/City-of-Helsinki/berth-reservations/blob/master/.github/workflows/review.yml#L72-L82

That would create just single job, so it can be seen that process works as should. It uses dynamic search of the secret, so it works in review environments.

Reply:
At least https://github.com/City-of-Helsinki/palvelutarjotin/blob/master/.github/workflows/production.yml works without any problem. Is there any huge difference?

Reply:
OK, I think it's fine then.